src/containers/Home/Home.test.js

```diff
@@ -24,20 +26,40 @@ const homeTestWrapper = () => {
 
 describe('Home', () => {
   it('should render the correct content', () => {
-    const { getByText, getByPlaceholderText, debug } = homeTestWrapper()
-
-
+    const { getByText, getByAltText, debug, getAllByRole } = homeTestWrapper()
+    //queryBy returns null if not found instead of failing
+    //findBy finds promises (combines getBy with waitFor)
+
+    const allImages = getAllByRole('img')
+    const sunImg = getByAltText('sun')
+    const titlePart2 = getByText('larizer')
+    const subtitle = getByText('Start Solarizing Now:')
     const addressLabel = getByText('Street Address')
-
-
```

Comment:
Examples of different methods to get items on the page.  Also a complete test of the home page